notification/package.json

```diff
@@ -11,16 +11,23 @@
         "test": "jest"
     },
     "dependencies": {
+        "@types/pg": "^7.14.3",
+        "amqplib": "^0.5.6",
         "axios": "^0.19.2",
         "body-parser": "^1.19.0",
         "cors": "^2.8.5",
+        "es6-shim": "^0.35.5",
```

Comment:
Can we instead add `"lib": ["es5"  , "dom"  ,  "es6"]` to `tsconfig.json` and remove the shim?